.travis.yml

```diff
@@ -33,13 +35,15 @@ env:
     - CONDA_CHANNEL_PRIORITY=True
     - NUMPY_VERSION=stable
 
+
+
 matrix:
   fast_finish: true
   include:
     - os : linux
       env: NAME='minimal'
            PYTHON_VERSION=2.7
-           SETUP_CMD='--with-memleak'
+           MAIN_CMD='python ./testsuite/MDAnalysisTests/mda_nosetests analysis auxiliary coordinates core formats topology --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50 --with-memleak && python ./testsuite/MDAnalysisTests/mda_nosetests utils --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50 --with-memleak'
```

Comment:
I prefer the logic of that!

I think the environment variable are set sequentially. Therefore you can still use `SETUP_CMD`. Also, you could have the list of tests for each call to nose in a couple of variable to avoid mistakes when a test will move from one run to the other.

Finally, it should not be `&&` between the two tests run, but only `;`. With `&&`, the second run only happens if the first one succeed.